<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
    <line>
      <![CDATA[STEP1]]>
    </line>
  </reporter-output>
  <suite started-at="2020-06-24T22:51:30 IST" name="Default suite" finished-at="2020-06-24T22:51:37 IST" duration-ms="7549">
    <groups>
    </groups>
    <test started-at="2020-06-24T22:51:30 IST" name="Default test" finished-at="2020-06-24T22:51:37 IST" duration-ms="7549">
      <class name="com.clearcode.testsuite.TestSuite1">
        <test-method is-config="true" signature="launch()[pri:0, instance:com.clearcode.testsuite.TestSuite1@29176cc1]" started-at="2020-06-24T22:51:30 IST" name="launch" finished-at="2020-06-24T22:51:37 IST" duration-ms="7144" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[STEP1]]>
            </line>
          </reporter-output>
        </test-method> <!-- launch -->
        <test-method signature="RealEstate(java.lang.String, java.lang.String, java.lang.String, [Ljava.lang.String;, java.lang.String)[pri:0, instance:com.clearcode.testsuite.TestSuite1@29176cc1]" started-at="2020-06-24T22:51:37 IST" name="RealEstate" data-provider="RealEstate" finished-at="2020-06-24T22:51:37 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:50)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:146)
at org.testng.internal.Parameters.handleParameters(Parameters.java:820)
at org.testng.internal.Parameters.handleParameters(Parameters.java:762)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:60)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:39)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:771)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
at com.clearcode.utils.ExcelUtils.getData(ExcelUtils.java:41)
at dataProvider.LoginData.RealEstate(LoginData.java:11)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:77)
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:46)
... 25 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RealEstate -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.clearcode.testsuite.TestSuite1@29176cc1]" started-at="2020-06-24T22:51:37 IST" name="close" finished-at="2020-06-24T22:51:37 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.clearcode.testsuite.TestSuite1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
